data/language/en-GB.txt

```diff
@@ -3830,3 +3830,5 @@ STR_6788    :Sensitivity:
 STR_6789    :Analogue stick sensitivity multiplier
 STR_6790    :Deadzone: {COMMA32}%
 STR_6791    :Sensitivity: {COMMA32}%
+STR_6792    :Entrances
```

Comment:
This is fine, but I wonder if 'Entrances buildings' would be better. What do you think?